### DISCUSSION TITLE
Expose ValueString as a proper list of maps

### DISCUSSION THREAD

**Discussion by gtirloni** [grafana/grafana#72636] [Alerting]
Only `ValueString` has the labels associated with a metric. We currently have to use regular expressions to extract that information needlessly.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82539. Feel free to continue the discussion around this there. Thank you!
